### DISCUSSION TITLE
support for websockets

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [thunderclient/thunder-client-support#881] [Ideas]
I don't postman it's a bloatware and takes lot of telemetry. Been using thunder client an year or so and I would really appreciate if it could support websocket requests like this: https://learning.postman.com/docs/sending-requests/websocket/websocket/

**Comment by jai-dewani** [comment]
Here to 2nd this request.
I always prefer using Thunder Client whenever possible, and would prefer using it for testing for WebSocket requests as well

**Comment by rangav** [comment]
Hi, We already implemented web sockets feature 
https://github.com/thunderclient/thunder-client-support/discussions/1643
